partner_priority/README.rst

```diff
@@ -0,0 +1,25 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
```

Comment:
LGPL --> AGPL on lines 1 and 3 still todo.

Reply:
Sorry I missed this, this has been fixed.